river/tree/isoup_tree_regressor.py

```diff
@@ -50,6 +50,21 @@ class iSOUPTreeRegressor(HoeffdingTreeRegressor, base.MultiOutputMixin):
     nominal_attributes
         List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
         should be treated as continuous.
+    attribute_observer
```

Comment:
Just marking them to review later.

Reply:
Updated in commit [f1eab02](https://github.com/creme-ml/creme/pull/379/commits/f1eab0274d21fedd3f599f8597e6a2f0230cbe26).